### PR TITLE
release 8.6 (reviewable changes)

### PR DESCRIPTION
**Pull request by pepopowitz**
Part of #4158. Targets the branch for #4420, and should be merged in there before #4420 is merged to main.

This is the reviewable PR for the 8.6 release! It does not include the massive amount of content changes, those are in #4420. 

This one _does_ include:

- chore: update versions in cutNewVersions.sh
- release: new version numbers
- release: new version numbers for content redirects
- release: new version numbers in check-versions workflow
- release: new versions for $/$ macros
- release: update version banners
- release: remove old versions from sitemap
- and a bunch of fixes for broken links and htaccess rules

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR) **This should go live by Tuesday morning US time, at latest.**
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

- [x] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process).

### CODE REVIEW

File: hacks/cutNewVersions.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ git commit -m "release: the big commit"
 
 # now start committing the small stuff
 # if there are uncommitted changes to this file, commit it:
-fi [ -n "$(git status --porcelain ./hacks/cutNewVersions.sh)" ]; then
```
Comment (pepopowitz) [inline hacks/cutNewVersions.sh:30]: Oops!

File: docusaurus.config.js [hunk 2]
```diff
@@ -64,13 +68,6 @@ module.exports = {
             label: "8.3 / 3.11.0",
             banner: "none",
           },
-          "3.10.0": {
-            banner: "none",
-          },
-          // surprising, yes, but true: 3.9 should show unsupported banner, but 3.7 should not.
-          "3.7.0": {
-            banner: "none",
-          },
```
Comment (akeller) [inline docusaurus.config.js:73]: 🎉 🧹

File: hacks/cutNewVersions.sh [hunk 3]
```diff
@@ -27,7 +27,7 @@ git commit -m "release: the big commit"
...
+if [ -n "$(git status --porcelain ./hacks/cutNewVersions.sh)" ]; then
   git add ./hacks/cutNewVersions.sh
   git commit -m "chore: update versions in cutNewVersions.sh"
 fi
```
Comment (akeller) [inline hacks/cutNewVersions.sh:33]: 👀 `if` or `fi`? on line 30 and 33?

Comment (pepopowitz) [reply hacks/cutNewVersions.sh:33]: yeah, I messed this up when I wrote it (or rather when I asked GitHub copilot to write it). `if` to start an if block; `fi` to end it.

**Review by akeller — APPROVED** [review]
Nothing exciting or spooky 👻 on the preview site!

**Pull request merged**